src/field_effect.c

```diff
@@ -237,7 +235,9 @@ static void Fldeff_MoveDeoxysRock_Step(u8 taskId);
 static IWRAM_DATA u8 sActiveList[32];
 
 // External declarations
-
+extern struct CompressedSpritePalette gMonPaletteTable[]; // GF made a mistake and did not extern it as const.
+extern const struct CompressedSpritePalette gTrainerFrontPicPaletteTable[];
+extern const struct CompressedSpriteSheet gTrainerFrontPicTable[];
```

Comment:
These two should come from `data.h`, instead of being extern'd

Reply:
I can't extern them, because data.h contains definition for gMonPaletteTable as const, and it can't be const, because one function won't match because of it.

Reply:
That's only for `gMonPaletteTable`.  The other two are const.